contrib/babelfishpg_common/src/varchar.c

```diff
@@ -95,7 +95,7 @@ is_basetype_nchar_nvarchar(Oid typid)
 int32_t
 GetUTF8CodePoint(const unsigned char *in, int len, int *consumed_p)
 {
-	int32_t		code;
+	int32_t		code = -1;
```

Comment:
I don't think this is required, if we are getting error we should add return of ereport or suppress warning